docs/testplan/Smartswitch-test-plan.md

```diff
@@ -645,6 +678,667 @@ root@sonic:/home/cisco# show chassis modules status
+
+#### Steps
+* Use command on DPU: `nohup bash -c "sleep 5 && echo c > /proc/sysrq-trigger" &`.
+* Powercycling of DPU is to ensure that pcie link came up properly after the memory exhaustion test.
+* Use command on Switch: `config chassis module shutdown <DPU_NUMBER>` to power off the DPUs.
+* Wait for 3 mins.
+* Use command on Switch: `config chassis module startup <DPU_NUMBER>` to power on the DPUs.
+        
+#### Verify in
+ * DPU and Switch
+   
+#### Sample Output
+
+DPU:
+
+```
+root@sonic:/home/admin# nohup bash -c "sleep 5 && echo c > /proc/sysrq-trigger" &
+.
+(Going for reboot)
+.
+```
+
+Switch: 
+
+```
+root@sonic:/home/cisco# show chassis modules status
+  Name           Description    Physical-Slot    Oper-Status    Admin-Status           Serial
+------  --------------------  ---------------  -------------  --------------  ---------------
+  DPU0  Data Processing Unit              N/A        Offline            down  154226463179136
+  DPU1  Data Processing Unit              N/A        Offline            down  154226463179152
+  DPU2  Data Processing Unit              N/A        Offline            down  154226463179168
+  DPUX  Data Processing Unit              N/A        Offline            down  154226463179184
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# config chassis startup DPU0
+root@sonic:/home/cisco# config chassis startup DPU1
+root@sonic:/home/cisco# config chassis startup DPU2
+root@sonic:/home/cisco# config chassis startup DPUX
+root@sonic:/home/cisco# 
+root@sonic:/home/cisco# show chassis modules status
+  Name           Description    Physical-Slot    Oper-Status    Admin-Status           Serial
+------  --------------------  ---------------  -------------  --------------  ---------------
+  DPU0  Data Processing Unit              N/A         Online              up  154226463179136
+  DPU1  Data Processing Unit              N/A         Online              up  154226463179152
+  DPU2  Data Processing Unit              N/A         Online              up  154226463179168
+  DPUX  Data Processing Unit              N/A         Online              up  154226463179184
+  
+root@sonic:/home/cisco# ping 169.254.200.1
+PING 169.254.200.1 (169.254.200.1) 56(84) bytes of data.
+64 bytes from 169.254.200.1: icmp_seq=1 ttl=64 time=0.160 ms
+^C
+--- 169.254.28.1 ping statistics ---
+1 packets transmitted, 1 received, 0% packet loss, time 0ms
+rtt min/avg/max/mdev = 0.160/0.160/0.160/0.000 ms
+root@sonic:/home/cisco#
+root@sonic:/home/cisco# show reboot-cause
+<cli implementation is in progress for getting the reasons in output for DPUs>
+```
+
+#### Pass/Fail Criteria
+ * Verify number of DPUs from inventory file for the testbed and number of DPUs shown in the cli output.
+ * Verify Ping works to all the mid plane ip listed in the ansible inventory file for the testbed.
+ * Verify `show reboot-cause <DPU_Number>` to check the reboot is caused by kernel panic. 
```

Comment:
Is this verified on DPU or Switch? could you prefix please?

Reply:
Its on switch.
Added the prefix
Not tested as it still in implementation